cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -538,12 +507,17 @@ instance (Show txid, Show tx)
       [ "kind" .= String "MsgKThxBye" ]
 
 
-instance Condense (HeaderHash blk)
+instance ConvertRawHash blk
       => ToObject (Point blk) where
-  toObject MinimalVerbosity p = toObject NormalVerbosity p
-  toObject verb p =
-    mkObject [ "kind" .= String "Tip" --TODO: why is this a Tip not a Point?
-             , "tip" .= showPoint verb p ]
+  toObject _verb GenesisPoint =
+    mkObject
+      [ "kind" .= String "GenesisPoint" ]
+  toObject verb (BlockPoint slot h) =
+    mkObject
+      [ "kind" .= String "BlockPoint"
+      , "slot" .= toJSON (unSlotNo slot)
```

Comment:
I _could_ use `ToObject SlotNo` here, but it feels odd to represent it as an object when its really just a sort of wrapper for a `Number`.